pkg/network/tracer/tracer_linux_test.go

```diff
@@ -992,23 +997,30 @@ func (s *TracerSuite) TestDNATIntraHostIntegration() {
 	require.NoError(t, err, "error connecting to client")
 	defer conn.Close()
 
-	_, err = conn.Write([]byte("ping"))
-	require.NoError(t, err, "error writing in client")
+	require.EventuallyWithT(t, func(c *assert.CollectT) {
+		_, err = conn.Write([]byte("ping"))
+		require.NoError(c, err, "error writing in client")
```

Comment:
FYI you cannot use require inside an EventuallyWithT.

Reply:
`CollectT` does implement `FailNow()`, so we should be able to use it with `require`.